### ISSUE TITLE
surfing and Journal of Sports Analytics

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/hyper2#105]
it would be good to rejig `surfing.Rmd`for submission to JSA.  The current one contains a hodgepodge of results and needs to be lean and mean before we can submit it.

**Comment by RobinHankin** [comment]
Submitted `surfing_jsa.pdf` [which is from `surfing_jsa.tex`] to JSA which is now MS596. Following email (to `hankin.robin@gmail.com`) received 30 August 2021:

Dear robin,

Thank you for your submission, MS 596, entitled

"Analysis of competitive surfing tournaments with reified Bradley-Terry likelihoods"

which was received on August 26, 2021. Eugene Shen has been
assigned as the coeditor in charge of your submission.

Please refer to the manuscript number, MS 596, in any future correspondence concerning your submission. You can contact Eugene Shen directly
by email at eugene.shen@gmail.com to find out the status of your submission.

You can also check the status of your submission at any time by visiting
the following url (if your email does not support built-in hyperlinks, copy
this url into your browser window and bookmark it, taking care not to split the
long url or allow intervening spaces: this url has a security password so that
only you will be able to access the status info on your submission).

https://editorialexpress.com/s.cgi?i=d3bda84f8fde2229e4f7c763bda1a567

Normally you should expect to wait about 12 weeks
before referees have returned reports and a decision can be made.

Sincerely,

Philip Maymin and Eugene Shen

**Comment by RobinHankin** [comment]
see e1fbdda

**Comment by RobinHankin** [comment]
See 0417704faad2c1905f9253e9d27fdeedeb819f83

**Comment by RobinHankin** [comment]
Will open a new issue specifically for the second referee's comment

**Comment by RobinHankin** [comment]
submitted a revised manuscript (anonymised), the result of `surfing_jsa.tex` with names removed, and also the rebuttal, the result of `surfing_jsa_rebuttal_referee2.tex`.  I received an acknowledgement of this from JSA, to my gmail account, just now

**Comment by RobinHankin** [comment]
Emailed Eugene Shen to enquire of the status of the manuscript:

Dear Professor Shen 

I am writing about manuscript JSA-596 "Analysis of competitive surfing tournaments..." by Driver and Hankin.  I tried to check its status on the JSA submission website, but it said that it did not recognise the submission.  Can you advise?  I worry that I have missed a communication from you; my last contact with you was in April this year.

Best wishes

Robin

**Comment by RobinHankin** [comment]
Eugene Shen replies via gmail, 3 August 2022:

I sent it back to the original reviewers.  Thanks!

**Comment by RobinHankin** [comment]
automatic email from JSA  [to gmail; 4 Aug 2022] reminding me that one of my attempted submissions did not work and they are going to delete it if I don't pick it up

**Comment by RobinHankin** [comment]
now published, closing issue.
